test/func/mocha.opts

```diff
@@ -0,0 +1,3 @@
+test/func/spec/**/*.spec.js
+--recursive
+--timeout 50000
```

Comment:
Do we really need 50sec test timeouts? Usually, only Sauce Labs would have this type of latency unless your webapp is huge / complex, which we arent...